addons/calendar/static/src/views/attendee_calendar/common/attendee_calendar_common_renderer.xml

```diff
@@ -6,7 +6,9 @@
                 <span t-if="isAlone" class="fa fa-exclamation-circle"/>
                 <t t-esc="title"/>
             </div>
-            <span t-if="!isTimeHidden" class="fc-time"/>
+            <t t-if="!isTimeHidden">
+                <span class="fc-time" t-esc="startTime" />
+            </t>
```

Comment:
```suggestion
            <span class="fc-time" t-esc="startTime" t-esc="startTime" />
```
this should be equivalent unless I'm missing something (I guess it was changed like this this to match `web.CalendarCommonRenderer.event`)